### PR TITLE
Hiding columns and keyboard labeling doc update

### PR DESCRIPTION
**Pull request by andrijapanicsb**
UI modification documentation

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
The new section here: https://acs-www.shapeblue.com/docs/pr58/adminguide/ui.html#modifying-the-ui

@nvazquez @anuragaw @shwstppr - kindly review and LGTM.

File: source/adminguide/ui.rst [hunk 1]
```diff
@@ -169,5 +169,93 @@ new, unique value.
...
+
+
+|add-instance-keyboards-customized.JPG|
+
+Notice short keyboard names (based on country code)
+
+
+|instance-metrics-customized-view.JPG|
+
+Notice missing "Zone" column
```
Comment (PaulAngus) [inline source/adminguide/ui.rst]:
```suggestion
Notice "Zone" column is hidden
```

File: source/adminguide/ui.rst [hunk 2]
```diff
@@ -169,5 +169,93 @@ new, unique value.
...
+
+
+|volume-metrics-customized-view.JPG|
+
+Notice missing "Utilisation" column
```
Comment (PaulAngus) [inline source/adminguide/ui.rst]:
```suggestion
Notice "Utilisation" column is hidden
```

**Comment by PaulAngus** [comment]
@nvazquez @anuragaw @shwstppr review technical correctness still required to merge and publish asap please

**Comment by PaulAngus** [comment]
requesting docbuild

**Comment by PaulAngus** [comment]
LGTM

**Pull request merged**